src/app/screens/LoginScreen/LoginForm.tsx

```diff
@@ -100,20 +95,12 @@ const LoginForm = ({signIn, isLoading}: Props) => {
         <Text style={styles.forgotPasswordText}>Forgot Password?</Text>
       </TouchableOpacity>
 
-      <TouchableOpacity
-        disabled={isLoading}
-        style={[
-          styles.loginButton,
-          isLoading ? styles.loginButtonDisabled : {},
-        ]}
-        activeOpacity={0.5}
-        onPress={handleSubmit(onSubmitHandler)}>
-        {isLoading ? (
-          <ActivityIndicator color="#ffffff" />
-        ) : (
-          <Text style={styles.loginButtonText}>Login</Text>
-        )}
-      </TouchableOpacity>
+      <Button
+        type={ButtonType.Primary}
```

Comment:
no need to import ButtonType and use it... we can directly gives type props as string like "primary", "secondary", ...